instrumentation/src/javatest/org/oppia/android/instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+"""
+This library contains the test suite source files of the instrumentation module.
+Note that:
+ - Each source file has its own kt_android_library target
+ - Target name of each kt_android_library as to be in small letters, separated by a underscore
```

Comment:
It seems that each test needs:
- A test binary
- A test library
- An instrumentation test

Could we fold these three into a single Bazel macro & helper .bzl file at the instrumentation/ level that we can use to more easily define end-to-end tests? That seems generally nice since we want to reduce the barrier to adding these tests as much as possible.

Reply:
According to the above comment if we avoid common deps for the test library we can't use kt_android_library in the wrapper.
For test binary and instrumentation test I've added them